### ISSUE TITLE
Suggest structure technical review and documentation based on WCAG Success criteria 

### ISSUE THREAD

**Issue by UsableNet-WCAG** [a11yedge/overlays#9]
Suggest structure technical review and documentation based on WCAG Success criteria .

As the discussion has identified, overlays use client-side code rewriting for automated remediation. To deploy a fix or remediation, a problem must first be identified, then new code generated to replace the existing code. This new code, along with the end-user experience, must be verified for both conformity with WCAG and, perhaps more importantly for complex websites, to ensure it does not break any other aspect or feature of the site.

For each WCAG Success Criterion, it is possible to identify a range of items that need to be checked, remediated, and verified. The second verification step above around safety —ensuring the site doesn’t break, bad code added —is unique to this client-side code update, as the original code has been altered outside the original websites process. 

Creating structured review of Accessibility at the edge  from WCAG, success criteria, testing, and remediation techniques would allow for a clear review of what can or should be automated, with or without the aid of AI.

Maybe to clarify if it needs some debate - All Accessibility issues can be resolved manually using client-side code updates - if you had time and resources you can even re-write the entire site. —manually meaning - using a regular development process of web accessibility testing methodology. Good accessibility developers (not easy to find) and manual verification of both the accessibility quality and overall website stability after the code update are essential. This requires effort, human resources, and good planning.

so Accessibility at the edge,  is about Understanding and documenting how much of this can be done or should be done automatically, suggest that structure could start with each WCAG Success Criterion, would be a good structure to follow.
